pkg/commands/compute/validate.go

```diff
@@ -77,8 +85,8 @@ type ValidateCommand struct {
 // NOTE: This function is also called by the `deploy` command.
 func validatePackageContent(pkgPath string) error {
 	files := map[string]bool{
-		"fastly.toml": false,
-		"main.wasm":   false,
+		manifest.Filename: false,
```

Comment:
Think I'm missing something, but how does manifest.Filename get set- it is a global variable?

Reply:
Correct! It's a global constant https://github.com/fastly/cli/blob/main/pkg/manifest/manifest.go#L9